lib/maru/entity.ex

```diff
@@ -173,41 +165,51 @@ defmodule Maru.Entity do
     Enum.map(attr_names, &do_expose(&1, options, do_func, __CALLER__))
   end
 
-  defp do_expose(attr_name, options, do_func, caller)
-    when is_atom(attr_name) and is_list(options) do
+  defp do_expose(attr_or_attrs, caller) do
+    quote bind_quoted: [
+      attr_or_attrs: attr_or_attrs,
+      caller:        Macro.escape(caller)
```

Comment:
Had to use `bind_quoted` because compiler complains when I use `unquote` inside the for loop. It says something like "You used `unquote` outside of a quoted expression".